.gitignore

```diff
@@ -20,3 +20,4 @@ temp/
 WriteAParquetFile.parquet
 .WriteAParquetFile.parquet.crc
 src/main/resources/NorthConcepts-DataPipeline.license.old
+src/main/resources/NorthConcepts-DataPipeline.license
```

Comment:
Hmmm...I thought I saw this change already. Do you need to pull master into your branch first?

BTW, it looks like you might have forked this repo and then are sending PRs from your master to the repo's master. This is okay, but we take different approach here. In the new approach you will clone this repo, create a branch (based on our naming pattern), commit your changes to the [feature] branch, and create a PR to merge your feature branch to master.

Here's a more detailed explanation:
https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow

Reply:
This was a different example from the first one. The file path was incorrect.
yes, I forked the repo...ill start using the new approach